src/Sentry/Checkin.cs

```diff
@@ -0,0 +1,72 @@
+namespace Sentry;
+
+/// <summary>
+/// Checkin metadata.
+/// </summary>
+public interface ICheckin
```

Comment:
Do we need an interface for this?

```suggestion
public interface ISentryCheckIn
```